fud/bitstream/gen_xo.tcl

```diff
@@ -1,10 +1,11 @@
-if { $::argc != 1 } {
+if { $::argc != 2 } {
     puts "ERROR: Program \"$::argv0\" requires 1 argument!\n"
```

Comment:
Should this be `requires 2 arguments!` now?

Can you add some context in comment form here for what these two arguments represent? Perhaps in the ERROR as well

Reply:
I missed that sorry. Will fix it soon.